### ISSUE TITLE
Failed opening 'Google/Client.php' 

### ISSUE THREAD

**Issue by jonatanaxe** [bluevisiontec/GoogleShoppingApi#9]
I am getting an error does anyone know what can it be?

2015-02-26T15:09:13+00:00 ERR (3): Warning: include(): Failed opening 'Google/Client.php' for inclusion (include_path='/var/www/vhosts/dev.store.com/app/code/local:/var/www/vhosts/dev.store.com/app/code/community:/var/www/vhosts/dev.store.com/app/code/core:/var/www/vhosts/dev.store.com/lib:.:/usr/share/pear:/usr/share/php')  in /var/www/vhosts/dev.store.com/lib/Varien/Autoload.php on line 93

**Comment by schnere** [comment]
Hi,

did you install the Google -ApiClient library?

For manual install you will have to download the Google ApiClient from here:
https://github.com/google/google-api-php-client

Place it under magento_root/vendor/google/apiclient

It should look like this:

```
$ ls -a vendor/google/apiclient/
.  ..  autoload.php  composer.json  CONTRIBUTING.md  examples  .gitignore  LICENSE  phpunit.xml.dist  README.md  src  style  tests  .travis.yml
```

The 'src' folder is the important one...

**Comment by jonatanaxe** [comment]
Yes
![ss](https://cloud.githubusercontent.com/assets/2486808/6394705/19e255ee-bdb2-11e4-8d15-f0ee6c1631b3.jpg)

**Comment by schnere** [comment]
Ah, yes. You do not get any error?
In this case you can ignore the warning. 

This is caused by ignoring the Magento Autoloader and using the autoloader of the Google ApiClient...

**Comment by JonatanM** [comment]
ok, when access the Manage Items I'm getting error, Error 500 Internal Server Error :(

**Comment by JonatanM** [comment]
Log

2015-02-26T15:42:35+00:00 ERR (3): Warning: include(Mage/Googleshopping/Helper/Data.php): failed to open stream: No such file or directory  in 
2015-02-26T15:42:40+00:00 ERR (3): Warning: include(): Failed opening 'Mage/Googleshopping/Helper/Data.php' for inclusion (include_path='/var/www/vhosts/dev.store.com/app/code/local:/var/www/vhosts/dev.store.com/app/code/community:/var/www/vhosts/dev.store.com/app/code/core:/var/www/vhosts/dev.store.com/lib:.:/usr/share/pear:/usr/share/php')  in /var/www/vhosts/dev.store.com/lib/Varien/Autoload.php on line 93

**Comment by jonatanaxe** [comment]
Thanks

**Comment by jonatanaxe** [comment]
The error was gone after upgrading it appeared

**Comment by jonatanaxe** [comment]
LOG

2015-03-02T13:27:49+00:00 ERR (3): Warning: include(): Failed opening 'Google/Client.php' for inclusion (include_path='/var/www/vhosts/dev.test.com/app/code/local:/var/www/vhosts/dev.test.com/app/code/community:/var/www/vhosts/dev.test.com/app/code/core:/var/www/vhosts/dev.test.com/lib:.:/usr/share/pear:/usr/share/php')  in /var/www/vhosts/dev.test.com/lib/Varien/Autoload.php on line 93
2015-03-02T13:27:51+00:00 ERR (3): Notice: Undefined index: files  in /var/www/vhosts/dev.test.com/app/code/core/Mage/Core/Model/Translate.php on line 138
2015-03-02T13:27:51+00:00 ERR (3): Warning: include(Google/Client.php): failed to open stream: No such file or directory  in /var/www/vhosts/dev.test.com/lib/Varien/Autoload.php on line 93
2015-03-02T13:27:51+00:00 ERR (3): Warning: include(): Failed opening 'Google/Client.php' for inclusion (include_path='/var/www/vhosts/dev.test.com/app/code/local:/var/www/vhosts/dev.test.com/app/code/community:/var/www/vhosts/dev.test.com/app/code/core:/var/www/vhosts/dev.test.com/lib:.:/usr/share/pear:/usr/share/php')  in /var/www/vhosts/dev.test.com/lib/Varien/Autoload.php on line 93

**Comment by schnere** [comment]
Sorry, this was my mistake. Should be fixed in 0.0.6

**Comment by jonatanaxe** [comment]
Thanks @schnere :)
